OneflowSDK.py

```diff
@@ -26,26 +31,30 @@ def requests_retry_session(self):
             connect=retries,
             backoff_factor=retryFactor,
             status_forcelist=retryStatus,
-            method_whitelist=retryMethods
+            allowed_methods=retryMethods,
         )
         adapter = HTTPAdapter(max_retries=retry)
-        session.mount('http://', adapter)
-        session.mount('https://', adapter)
+        session.mount("http://", adapter)
```

Comment:
very small, thing. adaptor can be spelled 2 ways but we agreed to 'o' previously to keep it consistent: https://oneflow.atlassian.net/wiki/spaces/DEV/pages/2097178/Adaptors